### ISSUE TITLE
The current master is failing in CI

### ISSUE THREAD

**Issue by limonte** [sweetalert2/sweetalert2-react-content#39]
https://travis-ci.org/sweetalert2/sweetalert2-react-content/builds/390371045

```
./node_modules/.bin/tsc src/sweetalert2-react-content.d.ts --lib dom,es6 

src/sweetalert2-react-content.d.ts:56:37 - error TS2344: Type 'keyof T' does not satisfy the constraint 'string'.
  Type 'string | number | symbol' is not assignable to type 'string'.
    Type 'number' is not assignable to type 'string'.

56 type Overwrite<T, U> = Pick<T, Diff<keyof T, keyof U>> & U;

```

@toverux would you mind to take a look on this issue?

**Comment by limonte** [comment]
@zenflow @toverux are you planning to maintain this project? If so, this issue should be fixed asap because:

- greenkeeper is useless - it creates a lot of falsy issues with warnings 
- travis is useless - every build will be red, which means every contribution will result into failure which means **poor DX experience**.  

Let's stay the outstanding example of how OSS should be made and not join the vast majority of repositories which were published and then abandoned ([Good Luck With That Public License](https://github.com/me-shaon/GLWTPL)).

**Comment by zenflow** [comment]
@limonte Yes, I do plan to maintain this project. 

*Excuse for being absent: My laptop died while I was travelling in South America. It's not an ideal place to buy a new laptop because of duties on imports, and also I've been doing travel things. To make matters worse I was robbed in Chile and lost my credit and debit card (making it difficult to make a large purchase).*

But as of yesterday, I'm in Buenos Aires in a new apartment with my girlfriend, for the next 2.5 months. 

So I aim to resolve this tomorrow.

**Comment by zenflow** [comment]
Because of this issue and #33, I was thinking of just removing TS support for now. 

I think the implementation is less than ideal right now anyways (just guessing), since it was done before the object-oriented refactor of SweetAlert2 and subsequently this library. I think it will have to be redone to be proper anyways. Sorry @toverux for not having the foresight about this.

If there's no objection to this then I'll make a PR tomorrow and leave it for a day or two to give @toverux a chance to intervene, though he seems to be busy.

**Comment by limonte** [comment]
I thank that TS is a must nowadays and its support is mandatory, let's not drop it. 

This should fix this issue https://github.com/sweetalert2/sweetalert2-react-content/pull/43

In order to fix #33 we need to either learn TS or wait for @toverux :) TS is in my learning roadmap, I'll try to do it asap.
